.github/workflows/macos.yml

```
@@ -71,9 +84,36 @@ jobs:
 
       - run: make
 
+      # Launchable requires Python and Java
```

I would suggest adding the "CLI" to clarify the "Launchable".

```suggestion
      # Launchable CLI requires Python and Java
```

---

Thanks. I confirmed it.